std/meta.d

```diff
@@ -1309,6 +1309,62 @@ private template staticMerge(alias cmp, int half, Seq...)
     }
 }
 
+/**
+ * Allows subsetting of alias sequence that are not
+ * necessarily adjacent to each other, i.e. cannot be sliced.
+ *
+ * Parameters:
+ *	range = a range object preferably an unsigned array to index T
+ *  T = a set of types to be indexed
+ *
+ * Returns: alias sequence of selected types
+ *
+ * Evaluates to `AliasSeq!(T[range[0]], T[range[1]], ..., T[range[n]])`.
+ *
+ * How:
+ * 		alias T1 = staticSubset!([0, 2, 1, 1, 2, 0], int, char, double);
+ */
+template staticSubset(alias range, T...)
+{
+	import std.range : isInputRange;
+	import std.traits : isArray, isNarrowString;
+
+	alias ArrT = typeof(range);
+	static if (isArray!ArrT && !isNarrowString!ArrT)
+	{
+		static if (range.length == 0)
+		{
+			alias staticSubset = AliasSeq!();
+		}
+		else
+		{
+			alias staticSubset = AliasSeq!(T[range[0]], staticSubset!(range[1 .. $], T));
+		}
+	}
+	else static if (isInputRange!ArrT)
+	{
+		import std.array : array;
+		alias staticSubset = staticSubset!(array(range));
+	}
+	else
+	{
+		static assert(false, "Cannot transform " ~ range.stringof ~ " of type " ~ ArrT.stringof ~ " into a AliasSeq.");
```

Comment:
No I edited the rebase file and manually resolved the conflicts in the meta.d file and then committed and pushed to my branch